MessageController.java

```diff
@@ -0,0 +1,58 @@
+package com.example.boilerscout.api;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import javafx.application.Application;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.DataAccessException;
+import org.springframework.http.HttpStatus;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+
+import javax.xml.bind.DatatypeConverter;
+import java.util.*;
+
+/*
+Created  by Baris Dingil
+ */
+
+@Service
+public class MessageController {
+    private static final Logger log = LoggerFactory.getLogger(Application.class);
+
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    public Map<String, Object> getMessage(@RequestBody Map<String, String> body){
```

Comment:
Rename this, `getMessage` implies something completely different from what this function is doing. Plus, this isn't even a `GET`, this is a `POST`